tests/worktree/worktree.c

```diff
@@ -610,3 +610,17 @@ void test_worktree_worktree__foreach_worktree_lists_all_worktrees(void)
 	int counter = 0;
 	cl_git_pass(git_repository_foreach_worktree(fixture.repo, foreach_worktree_cb, &counter));
 }
+
+void test_worktree_worktree__validate_invalid_worktreedir(void)
+{
+	git_worktree *wt;
+
+	cl_git_pass(git_worktree_lookup(&wt, fixture.repo, "testrepo-worktree"));
+	git__free(wt->worktree_path);
+	wt->worktree_path = "/path/to/invalid/worktreedir";
```

Comment:
This feels a little bit like too much magic to me.  😀  Is there a way that we can invalidate the work tree?  If I’m reading this correctly, we can just rm the work tree itself which would serve the same purpose but test the intent more directly and without having to change the innards of the work tree object.